sam/methods.go

```diff
@@ -56,19 +56,21 @@ func (s SamSlice) Write(filename string) { // Does not write header
 	}
 }
 
-func (s *SamAln) WriteToFileHandle(file *fileio.EasyWriter) {
-	WriteAlnToFileHandle(file, s)
+func (s *Sam) WriteToFileHandle(file *fileio.EasyWriter) {
+	WriteToFileHandle(file, *s)
 }
 
-func (s *SamAln) NextRealRecord(file *fileio.EasyReader) bool {
+func (s *Sam) NextRealRecord(file *fileio.EasyReader) bool {
 	var done bool
-	var next *SamAln
+	var next *Sam
+	var curr Sam
 	for nextBytes, err := file.Peek(1); next == nil && !done; nextBytes, err = file.Peek(1) {
 		if err == nil && nextBytes[0] == '@' {
 			fileio.EasyNextLine(file)
 			continue
 		}
-		next, done = NextAlignment(file)
+		curr, done = ReadNext(file)
```

Comment:
I am thinking that ReadNext already has the logic to skip the header and just give us the next alignment no matter where the file handler is in the file.  Can't we just call ReadNext(file) here and avoid the whole nextBytes for loop?

Reply:
Yes. This change has been made.